bin/rucio-hermes

```diff
@@ -20,6 +20,7 @@
 # - Hannes Hansen <hannes.jakob.hansen@cern.ch>, 2018
 # - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2020
 # - David Población Criado <david.poblacion.criado@cern.ch>, 2021
+# - Joel Dierkes <joel.dierkes@cern.ch>, 2021
```

Comment:
This PR has a lot of files that are changed and from a quick overview, I see that a lot of these changes include @joeldierkes as an author, like here. Which makes sense as the `add-header.py` is essentially looking at the commit history. I believe you can run the `add-header.py` and offset the git log to `HEAD~1` in order to keep the header consistent with the code contributors. ;)  Maybe it also brings down the number of files that need to be reviewed here significantly.

Reply:
Yeah my bad, I forgot to use the already implemented `--disable-add-me` argument.

Reply:
Maybe it's better to have a "lazy" approach to enforcing header validation? Instead of verifying the whole project's headers, only run the verification on files which are modified by the Push ? 
It avoids this "fix everything commit"; will gradually improve the headers during small commits. 

Just an idea, how it's done righ now is ok for me too.

Reply:
I tried that at first, but there are some limitations. I ran into the same problems as this Community post: https://github.community/t/can-i-process-only-changed-files-with-github-actions/137814/6 (under the __`//edit`__ part with the rewrite commits). At that moment, I decided to switch the strategy, since I didn't want to waste more time.

I talked with Mayank about it and we both decided, that this approach is fine, too.

Reply:
Thank you for the link. As I said, I'm ok to do it this way.

Reply:
Okay perfect, I'll create a second version to Review then.

Also, Mayank suggested to only check the changes on a PR, and not on every push. I will implement that too.